src/shared/components/table/Table.scss

```diff
@@ -0,0 +1,17 @@
+@import 'styles/mixins';
+
+.table {
```

Comment:
This component is ui-based component - should be placed in UI dir based on conventions